### ISSUE TITLE
remove too much loss and model

### ISSUE THREAD

**Issue by LittleFish-Coder** [LittleFish-Coder/GemGNN#34]
in the thesis root/thesis/context/
- methodology
- results
- experiment

Check if we overthinking and over-write too much describing the model and loss

currently, I found the best loss function is plain cross entropy which means only use the --loss_fn ce in train_hetero_graph.py

so please just write about the crossEntropy method in it, remove the redundant parts

and I only use HAN([Heterogeneous Graph Attention Network](https://arxiv.org/abs/1903.07293)) for training.

so remove these part if needed.

check all the session in my paper

make sure to trationalize why I pick this loss function and model

I know HAN seems not very appropriate but please do as well as you can
